docs/maui/converters/item-tapped-eventargs-converter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: ItemTappedEventArgsConverter - .NET MAUI Community Toolkit
+author: cliffagius
+description: "The ItemTappedEventArgsConverter is a converter that allows users to extract the Item value from an ItemTappedEventArgs object."
+ms.date: 04/13/2022
+---
+
+# ItemTappedEventArgsConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `ItemTappedEventArgsConverter` is a converter that allows users to extract the Item value from an `ItemTappedEventArgs` object. It can subsequently be used in combination with EventToCommandBehavior.
```

Comment:
```suggestion
The `ItemTappedEventArgsConverter` is a converter that allows users to extract the Item value from an `ItemTappedEventArgs` object. It can subsequently be used in combination with `EventToCommandBehavior`.
```

We should probably link this in the future when we have the Docs for the EventToCommandBehavior

Reply:
Added the link but it will fail at present until the docs are added but will add them to my todo list.